src/components/layouts/collection-page-layout.tsx

```diff
@@ -736,4 +743,47 @@ const Fresh = ({freshness}: {freshness: any}) => {
   )
 }
 
+const CoursePodcast = ({
+  simplecast_uid: id,
+  transcript,
+  instructorName,
+}: any) => {
+  const [isOpen, setOpen] = React.useState(false)
+
+  if (isEmpty(id)) {
+    return null
+  } else {
+    return (
+      <div className="w-100 pt2 pb3 base-secondary ph0-ns ph2">
+        <h3 className="mt2">
+          {`Listen to ${instructorName} tell you about this course.`}{' '}
+          {transcript && (
+            <span>
+              <button
+                className="link pointer bn bg-transparent f5 fw4 blue-secondary"
+                onClick={() => setOpen(!isOpen)}
+              >
+                {isOpen ? 'Hide Transcript' : 'Show Transcript'}
+              </button>
+            </span>
+          )}
+        </h3>
+        <iframe
+          height="52px"
+          width="100%"
+          frameBorder="no"
+          scrolling="no"
+          seamless
+          src={`https://player.simplecast.com/${id}?dark=false`}
```

Comment:
darkmode always turned off but I tried it out and it's set as black so it looks weird anyway.

Reply:
you can specify color with `&color=111827` and check for color mode with `{useTheme} from 'next-themes'`

Reply:
oh thanks @vojtaholik, I'll check that out